sdk/testnet_consts.go

```diff
@@ -100,6 +100,11 @@ var KnownTestnetAutomaticRelayerEmitters = []struct {
 	{ChainId: vaa.ChainIDArbitrum, Addr: "000000000000000000000000Ad753479354283eEE1b86c9470c84D42f229FF43"},
 	{ChainId: vaa.ChainIDOptimism, Addr: "00000000000000000000000001A957A525a5b7A72808bA9D10c389674E459891"},
 	{ChainId: vaa.ChainIDBase, Addr: "000000000000000000000000ea8029CD7FCAEFFcD1F53686430Db0Fc8ed384E1"},
+	{ChainId: vaa.ChainIDBerachain, Addr: "000000000000000000000000362fca37E45fe1096b42021b543f462D49a5C8df"},
```

Comment:
I don't see a contract at this address. https://testnet.berascan.com/address/0x362fca37E45fe1096b42021b543f462D49a5C8df